src/client/Microsoft.Identity.Client/Internal/JsonWebToken.cs

```diff
@@ -140,6 +140,7 @@ public JWTHeader(X509Certificate2 certificate)
                 Certificate = certificate;
             }
 
+            [JsonIgnore]
             protected X509Certificate2 Certificate { get; }
```

Comment:
SJT is trying to source gen this and this causes a bunch of problems. We don't serialize the cert.